### DISCUSSION TITLE
Slight Change to the UI

### DISCUSSION THREAD

**Discussion by akeljo** [Novik/ruTorrent#2515] [Q&A] [answered]
Where would I start to change the side menu to look like the image?
![rut1](https://github.com/Novik/ruTorrent/assets/7863563/f8b10b04-df99-4218-bbb2-9ba81fcf3e1f)
![rut2](https://github.com/Novik/ruTorrent/assets/7863563/d6038011-9cb1-48af-a60a-7d73e4fe2c81)

**Comment by stickz** [comment]
I would try out different themes in `Settings -> General`. The gear icon at the top will open the settings menu.

**Comment by Novik** [comment, marked as answer]
![2023-05-24_21h02_26](https://github.com/Novik/ruTorrent/assets/1230409/e9232266-9da3-4aac-84ce-723a486963ef)

**Reply by akeljo** [reply]
A big thank you
